### PR TITLE
Feature/cross modality new costs

### PR DESCRIPTION
**Pull request by nosander**

### CODE REVIEW

File: src/moscot/base/problems/problem.py [hunk 1]
```diff
@@ -102,11 +102,11 @@ def _split_mass(arr: ArrayLike) -> ArrayLike:
...
             raise ValueError(f"Expected array of shape `({adata.n_obs,}, ...)`, found `{data.shape}`.")
-        if np.any(data < 0.0):
-            raise ValueError("Some entries have negative mass.")
+        # if np.any(data < 0.0):
+        #    raise ValueError("Some entries have negative mass.")
         total = np.sum(data, axis=0, keepdims=True)
-        if np.any(total <= 0.0):
-            raise ValueError("Some measures have no mass.")
+        # if np.any(total <= 0.0):
+        #    raise ValueError("Some measures have no mass.")
```
Comment (nosander) [inline src/moscot/base/problems/problem.py:109]: how are we handling this?

File: src/moscot/datasets.py [hunk 2]
```diff
@@ -222,6 +222,56 @@ def sim_align(
...
+    -------
+    Annotated data object.
+    """
+    return _load_dataset_from_url(
+        path, backup_url="https://figshare.com/ndownloader/files/40047034", expected_shape=(6224, 8000), **kwargs
+    )
+
+
+def bone_marrow_rna(
+    path: PathLike = "~/.cache/moscot/bone_marrow_rna.h5ad",
```
Comment (nosander) [inline src/moscot/datasets.py:251]: Re-Uplaod didn't work..?!
Cell_types are still NaN

**Comment by nosander** [comment]
Tutorial: https://github.com/nosander/moscot_notebooks/blob/feature/tutorialTranslationProblem/tutorials/600_tutorial_translation.ipynb

File: docs/references.bib [hunk 3]
```diff
@@ -293,3 +293,120 @@ @article{spanjaard:18
...
+
+@article{lange-moslin:23,
+  author    = {Lange, Marius and Piran, Zoe and Klein, Michal and Spanjaard, Bastiaan and Klein, Dominik and Junker, Jan Philipp and Theis, Fabian J. and Nitzan, Mor},
+  publisher = {Cold Spring Harbor Laboratory},
+  url       = {TODO},
+  doi       = {TODO},
+  journal   = {bioRxiv},
+  title     = {Mapping lineage-traced cells across time points with moslin},
+  year      = {2023},
+}
```
Comment (nosander) [inline docs/references.bib:412]: to do: add scot reference? How can I avoid merging conflict?

File: src/moscot/datasets.py [hunk 4]
```diff
@@ -171,18 +180,63 @@ def zebrafish(
...
+        return _load_dataset_from_url(
+            path + "_rna.h5ad",
+            backup_url="https://figshare.com/ndownloader/files/40046950",
+            expected_shape=(6224, 2000),
+            **kwargs,
+        )
+    return _load_dataset_from_url(
+        path + "_atac.h5ad",
+        backup_url="https://figshare.com/ndownloader/files/40047034",
+        expected_shape=(6224, 8000),
```
Comment (nosander) [inline src/moscot/datasets.py:219]: Re-Uplaod didn't work..?!
Cell_types are still NaN

File: src/moscot/base/problems/problem.py (hunk 1 above)
Comment (MUCDK) [reply src/moscot/base/problems/problem.py:109]: Add `# TODO(@MUCDK, @michalk): possibly introduce class property indicating whether we should check for negative mass` and leave it commented out, i.e.leave it as it is and only add this comment :) We will discuss this in the PR to the moscot repo.

File: docs/references.bib (hunk 3 above)
Comment (MUCDK) [reply docs/references.bib:412]: what merge conflict do you have?

Comment (MUCDK) [reply docs/references.bib:412]: also, please run `tox -e format-references`

**Pull request closed without merging**